### PR TITLE
Add z-index utility to docs

### PR DESCRIPTION
**Pull request by tedk13**
Issue: https://github.com/ritterim/platform-ui/issues/722

Screen Shot 2022-08-02 at 11 29 49 AM
Screen Shot 2022-08-02 at 11 29 56 AM

### CODE REVIEW

File: content/docs/utilities/z-index.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+title: Z-Index
+skellyCSS: true
+---
+
+{{% anchor name="Z-Index" %}}
+
+You can use these utility classes to control the stack. Add a specified `z-{value}` class to your element to change the order of the stack. Remember to specify at least `position: relative` on the element for `z-index` to work.
+
+{{< classes result="true" >}}
+{{< classes-row class="z-0" result="z-index: 0" >}}
```
Comment (jaime-lynn) [inline content/docs/utilities/z-index.md:11]: I'd put these results so they spit out `z-index: 0` instead of z-index: 0

Comment (tedk13) [reply content/docs/utilities/z-index.md:11]: Oh you mean put the little code ticks around them?

File: content/docs/utilities/z-index.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
+---
+title: Z-Index
+skellyCSS: true
+---
+
+{{% anchor name="Z-Index" %}}
+
+You can use these utility classes to control the stack. Add a specified `z-{value}` class to your element to change the order of the stack. Remember to specify at least `position: relative` on the element for `z-index` to work.
```
Comment (jaime-lynn) [inline content/docs/utilities/z-index.md:8]:
```suggestion
You can use these utility classes to control the stack. Add a `z-{value}` class to your element to change the order of the stack. Remember to specify `position: relative` on the element for `z-index` to work.
```

Is there another term other than "the stack"? It's weird to me to have that repeated

**Comment by jaime-lynn** [comment]
The `data-z` thing is wicked cool!

File: content/docs/utilities/z-index.md (hunk 2 above)
Comment (tedk13) [reply content/docs/utilities/z-index.md]: Not sure...

**Comment by tedk13** [comment]
`data-z` not my idea but I'll take the credit.

File: content/docs/utilities/z-index.md (hunk 1 above)
Comment (tedk13) [reply content/docs/utilities/z-index.md:11]: Screen Shot 2022-08-02 at 11 50 36 AM

File: content/docs/utilities/z-index.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
+---
+title: Z-Index
+skellyCSS: true
+---
+
+{{% anchor name="Z-Index" %}}
+
+You can use these utility classes to control the stack. Add a `z-{value}` class to your element to change the order of the stack. Remember to specify `position: relative` on the element for `z-index` to work.
```
Comment (jaime-lynn) [inline content/docs/utilities/z-index.md:8]:
```suggestion
You can use these utility classes to control the stacking context. Add a `z-{value}` class to your element to change the order of the stack. Remember to specify `position: relative` on the element for `z-index` to work.
```

File: content/docs/utilities/z-index.md (hunk 2 above)
Comment (mrtrimble) [reply content/docs/utilities/z-index.md]: maybe something like:

> You can use these utility classes to control **elements layered in a stack**. Add a `z-{value}` class to your element to change the order of the stack.`

Explains what "the stack" is referencing.

File: content/docs/utilities/z-index.md (hunk 3 above)
Comment (hougasian) [reply content/docs/utilities/z-index.md]: These utility classes control the stacking order. Adding a `z-{value}` class to your element will change its order in the stack. Remember to add the `pos-rel` class on the element for `z-index` to work.

Comment (tedk13) [reply content/docs/utilities/z-index.md]: I wasn't sure about explicitly saying `use pos-rel` because the element could be `fixed` or `absolute` too

Comment (hougasian) [reply content/docs/utilities/z-index.md]: then remove the sentence entirely. 
PlatformUI disclaimer: Platform UI utilities assume a basic knowledge of CSS and the DOM.

Comment (tedk13) [reply content/docs/utilities/z-index.md]: Roger that

**Review by hougasian — APPROVED** [review]

**Pull request merged**